### ISSUE TITLE
Widget behind Clock

### ISSUE THREAD

**Issue by vonlocklearj** [TheLumbee/qt-digital-clock#2]
Since we are not using a main window we need some type of widget behind the Clock so we can see the numbers on from clock.

**Comment by Chavan4** [comment]
I agree as well, what widget should we use and why? @TheLumbee @vonlocklearj 
I propose a widget that has a white background.

**Comment by TheLumbee** [comment]
How about this?

https://doc.qt.io/qt-6/qwidget.html

**Comment by Chavan4** [comment]
Looks good to me, i agree to using it. @TheLumbee @vonlocklearj

**Comment by vonlocklearj** [comment]
After looking into this I agree as well. This will work great for the Clock. Thank you @TheLumbee.
